docs/PYPI.md

```diff
@@ -165,9 +169,6 @@ import logging
 logging.basicConfig(level=logging.DEBUG)
 ```
 
-**Note:** _A lot of printing can happen, including sensitive information, so
-make sure to be aware of what you're doing if you're enabling it!_
-
 #### Middleware
 
 _From version 0.4.0-dev, the middleware system has been introduced. This system
```

Comment:
For some reason I can't select all lines here so I'll just paste my suggestion
```markdown
The middleware system was introduced in version `0.4.0-dev`. This system gives you the
freedom to create custom events and remove the already existing middleware created by
the developers. Your custom middleware directly receives the payload from
Discord. You can't do anything wrong without accessing the `override` attribute, but if
you do access it, the Pincer team will not provide any support for weird behavior.
So, in short, only use this if you know what you're doing. An example of using
the middleware system (with a custom `on_ready` event) can be found
[in our docs](https://pincer.readthedocs.io/en/latest/pincer.html#pincer.client.middleware).
```

Reply:
I like this except for `on_ready` being in parenthesis so I won't keep that

Reply:
Sure